WordPress/src/main/java/org/wordpress/android/ui/quickstart/QuickStartMySitePrompts.kt

```diff
@@ -11,44 +11,58 @@ enum class QuickStartMySitePrompts constructor(
     val parentContainerId: Int,
     val focusedContainerId: Int,
     val shortMessagePrompt: Int,
-    val iconId: Int
+    val iconId: Int,
+    val promptDialogTitleId: Int,
+    val promptDialogMessageId: Int
 ) {
     VIEW_SITE_TUTORIAL(
             QuickStartTask.VIEW_SITE,
             R.id.my_site_scroll_view_root,
             R.id.row_view_site,
             R.string.quick_start_dialog_view_site_message_short,
```

Comment:
I removed `tools:ignore="UnusedResources"` from the majority of strings, but `lint` is still angry at the rest, maybe because they are in enum.. Any idea why it might be?